pyaerocom/aeroval/coldatatojson_engine.py

```diff
@@ -615,9 +619,132 @@ def _get_statistics(obs_vals, mod_vals, min_num):
                             min_num_valid=min_num)
     return _prep_stats_json(stats)
 
+def _make_trends_from_timeseries(obs, mod, freq, season, start, stop, min_yrs):
+    """
+    Function for generating trends from timeseries and fomatting it in a way
+    that can be serialized to json. A key, map_var, is added
+    for use in the web interface.
+
+    Parameters
+    ----------
+    obs     : pd.Series
+        Time series of the obs
+    mod     : pd.Series
+        Time series of the mod
+    freq    : str
+        Frequency for the trends, either monthly or yearly
+    season  : str
+        Seasons used for the trends
+    start   : int
+        Start year
+    stop    : int
+        Stop year
+    min_yrs : int
+        Minimal number of years for the calculation of the trends
+
+    Raises
+    ------
+    AeroValTrendsError
+        If stop - start is smaller than min_yrs
+    
+    AeroValError
+        If the trend engine returns None
+
+    Returns
+    ------
+    (dict, dict)
+        Dicts consiting of the trends data for the obs and mod
+    """
+
+
+    if stop-start < min_yrs:
+        raise AeroValTrendsError(f"min_yrs ({min_yrs}) larger than time between start and stop")
+
+    te = TrendsEngine
+
+
+    # The model and observation data are made to pandas times series
+    obs_trend_series = obs
+    mod_trend_series = mod
+
+    # Translate season to names used in trends_helpers.py. Should be handled there instead!
+    season = _get_season_from_months(season)
+
+    # Trends are calculated
+    obs_trend = te.compute_trend(obs_trend_series, freq, start, stop, 
+                                                                min_yrs, season)
+    mod_trend = te.compute_trend(mod_trend_series, freq, start, stop, 
+                                                                min_yrs, season)
+
+    # Makes pd.Series serializable
+    if obs_trend["data"] is None or mod_trend["data"] is None:
+        raise AeroValTrendsError("Trends came back as None",
+                                     obs_trend["data"], mod_trend["data"])
+
+    obs_trend["data"] = obs_trend["data"].to_json()
+    mod_trend["data"] = mod_trend["data"].to_json()
+
+    obs_trend["map_var"] = "slp_2000"
+    mod_trend["map_var"] = "slp_2000"
+
+    return obs_trend, mod_trend
+
+def _make_trends(obs_vals, mod_vals, time, freq, season, start, stop, min_yrs):
+    """
+    Function for generating trends from timeseries and fomatting it in a way
+    that can be serialized to json. This will calculate pandas time series
+    from the lists and use that to calculate trends
```

Comment:
```suggestion
    Function for generating trends from timeseries and fomatting it in a way
    that can be serialized to json. This will calculate pandas time series
    from the lists and use that to calculate trends
```
```suggestion
    Function for generating trends from timeseries 
    
    Includes fomatting it in a way
    that can be serialized to json. This will calculate pandas time series
    from the lists and use that to calculate trends
```

There is always a one-liner for each docstring in the beginning :)